tutor/src/components/annotations/annotation.jsx

```diff
@@ -36,6 +36,26 @@ function getSelectionRect(win, selection) {
   };
 }
 
+// modified copy/paste out of 'serialize-selection' module
+function serializeSelectionSave(referenceNode, range) {
+  referenceNode = referenceNode || document.body
+
+  const cloneRange = range.cloneRange();
+  const startContainer = cloneRange.startContainer;
+  const startOffset = cloneRange.startOffset;
+  const state = { content: cloneRange.toString() };
+
+  cloneRange.selectNodeContents(referenceNode);
+  cloneRange.setEnd(startContainer, startOffset);
+
+  state.start = cloneRange.toString().length;
+  state.end = cloneRange.toString().length + state.content.length;
+
+  state.restore = serializeSelection.restore.bind(null, state, referenceNode);
+
```

Comment:
was `bind` used before?  I'm not sure if we have a polyfill loaded for that.  If it wasn't it might be better to use a fn, i.e. 

`state.restore = () => serializeSelection.restore(state, referenceNode)`

Reply:
@nathanstitt that function is mostly copy/paste out of the `serialize-selection` library with the one change that you can pass the range in instead of it grabbing it from `window.getSelection().getRangeAt(0)`, i was leaning towards changing the code as little as possible but i can clean it up if you want

Reply:
yep, I see the `bind` in serialize lib now, I'd missed it before I guess